src/apis/comment/useGetCommentList.ts

```diff
@@ -0,0 +1,31 @@
+import { axiosInstance } from '../axios';
+import type { CommentListResponse } from '../../types/comment';
+import type { CategoryType } from '../../types/comment';
+import { useQuery } from '@tanstack/react-query';
+import { queryKey } from '../../constants/queryKey';
+
+// 댓글 조회
+export const getCommentList = async (
+  targetId: number,
+  category: CategoryType
+): Promise<CommentListResponse> => {
+  try {
+    const response = await axiosInstance.get('/api/comments', {
+      params: {
+        targetId,
+        category,
+      },
+    });
+    return response.data.result;
+  } catch (error) {
+    console.error('댓글 조회 실패', error);
+    throw error;
+  }
+};
+
+export const useGetCommentList = (targetId: number, category: CategoryType) => {
+  return useQuery({
+    queryKey: [queryKey.COMMENT_LIST, targetId, category],
```

Comment:
```suggestion
    queryKey: [queryKey.COMMENT_LIST, {targetId, category}],
```
P1: 괄호를 붙여주면 targetId, category 순서와 무관하게 일관성 있게 쿼리키를 관리할 수 있습니답

Reply:
헉 그렇군요 너무 유용하네요 감사합니다!! 커밋 반영하였습니다.